.github/workflows/push-docker-image.yml

```diff
@@ -28,37 +26,34 @@ jobs:
             return tagName;
           result-encoding: string
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-        
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
 
       - name: Login to DockerHub
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
-           username: ${{ secrets.DOCKERHUB_USERNAME }}
-           password: ${{ secrets.DOCKERHUB_PASSWORD }}
-     
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
       - name: Build and push Dev Docker image
-        uses: docker/build-push-action@v4
-        
+        uses: docker/build-push-action@v5
         with:
           push: true
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_PASSWORD }}
           target: dev
-          repository: openedx/edx-notes-api-dev
-          tags: ${{ steps.get-tag-name.outputs.result }},${{ github.sha }}
+          tags: |
+            openedx/edx-notes-api-dev:${{ steps.get-tag-name.outputs.result }}
+            openedx/edx-notes-api-dev:${{ github.sha }}
           platforms: linux/amd64,linux/arm64
 
-      # - name: Build and push prod Docker image
-      #   uses: docker/build-push-action@v4
-      #   with:
-      #     push: true
-      #     username: ${{ secrets.DOCKERHUB_USERNAME }}
-      #     password: ${{ secrets.DOCKERHUB_PASSWORD }}
-      #     target: prod
-      #     repository: openedx/edx-notes-api
-      #     tags: ${{ steps.get-tag-name.outputs.result }},${{ github.sha }}
-      #     platforms: linux/amd64,linux/arm64
\ No newline at end of file
+      - name: Build and push Prod Docker image
```

Comment:
@feanil 
Prod images were commented in this [PR](https://github.com/openedx/edx-notes-api/pull/320/files#diff-17cd2e03fb1dfb62371b2743de71db3bf8e2ab0a5fd0016ae5f8dc31ee690fd8R41
)
Do you have information we did we do that?
Seems ok to me to uncomment it.

Reply:
If we're not pushing the images, then we don't need this workflow at all.  Let's just remove it, it was 2U specific anyway.